main.go

```diff
@@ -58,7 +64,6 @@ func Main() {
 	// Initialize mDNS
 	if err := initMdns(); err != nil {
 		logger.Error().Err(err).Msg("failed to initialize mDNS")
-		os.Exit(1)
 	}
```

Comment:
// TODO: disable/reset mDNS option?

Reply:
We perform a non-nil check for each call to mdns, so it is not necessary to do so now.

Reply:
I actually meant (and communicated very poorly, sorry), do we need to _reset to default_ the mDNS configuration or disable mDNS here, like we indicated we need eventually `// TODO` on line 56.